drivers/spi/spi_ll_stm32.c

```diff
@@ -90,7 +90,7 @@ static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
 	u16_t rx_frame;
 
 	tx_frame = spi_stm32_next_tx(data);
-	while (!LL_SPI_IsActiveFlag_TXE(spi)) {
+	while (!spi_stm32_ll_func_IsActiveFlag_TXE(spi)) {
```

Comment:
New functions, which are zephyr specific should follow zephyr coding style, hence should be lower_case (not CamelCase)